dotnet/src/Extensions/TemplateEngine.PromptTemplateEngine/Blocks/Block.cs

```diff
@@ -41,4 +41,24 @@ private protected Block(string? content, ILoggerFactory? loggerFactory)
     /// <param name="errorMsg">Error message in case the content is not valid</param>
     /// <returns>True if the block content is valid</returns>
     public abstract bool IsValid(out string errorMsg);
+
+    internal static bool IsVarPrefix(char c)
```

Comment:
Would it be better if we put these helper methods to static `Symbols` class?

Reply:
+1 for moving these methods to the `Symbols` class. The `Block` class should not change every time a new symbol or helper method is added, removed, or changed.